src/OrchardCore/OrchardCore.ResourceManagement.Abstractions/ResourceManagementOptions.cs

```diff
@@ -1,22 +1,20 @@
-using System;
-
 namespace OrchardCore.ResourceManagement
 {
     public class ResourceManagementOptions
     {
-        public bool UseCdn { get; set; } = true;
+        public bool UseCdn { get; set; }
```

Comment:
I suggest to leave the default values here, and use these default in the `SiteSettings`

Reply:
Hmm, I think will apply default values to both places, but without a dependency. makes sense